assets/js/projectComponents/pages/routing/index.js

```diff
@@ -0,0 +1,34 @@
+import api from '../api/index'
+import store from '~/store'
+
+const loadPageBeforeEnter = async (to, from, next) => {
+  store.dispatch('startFetching')
```

Comment:
I think you don't need to handle the `*Fetching` actions. If the route has the before enter it should handle it properly:
https://github.com/uc-group/bear-it/blob/pg-pages/assets/js/plugins/projectComponents.js#L19-L41